qt/python/mantidqt/utils/qt/plugins.py.in

```diff
@@ -0,0 +1,57 @@
+#  This file is part of the mantid workbench.
+#
+#  Copyright (C) 2017 mantidproject
+#
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""Defines functions to setup paths to Qt plugins at runtime. This is
+a generated file. Any changes made here will be lost. See
+qt/python/mantidqt/utils/qt/plugins.py.in
+"""
+from __future__ import absolute_import
+
+# stdlib imports
+import os
+
+# 3rd-party imports
+from qtpy import QT_VERSION
+from qtpy.QtCore import QCoreApplication
+
+
+# Qt needs to be able to find its plugins and by default it looks in the
+# directories where they were built. On Linux the libraries are in fixed
+# locations.
+#
+#  On Windows the package can be installed anywhere so the locations must
+#  be determined at runtime. The two options are:
+#    1. using a qt.conf file: requires it to be next to the application
+#       executable but this would be python.exe and then would not allow
+#       switching between Qt4/Qt5
+#    2. add extra paths to the LibraryPath list here
+# As we can dynamically determine the Qt version here we choose option 2.
+
+def setup_library_paths_win():
+    """Adds the build-time configured directory to the Qt library path.
+    The buildsystem generates the path at build time.
+    A %V marker can be used that will be replaced by the major version of Qt
+    at runtime.
+    """
+    QCoreApplication.addLibraryPath(
+        '@QT_PLUGINS_PATH@'.replace('%V', QT_VERSION[0])
+    )
+
+# Set the implementation appropriate for the platform
+if os.name == 'nt':
+    setup_library_paths = setup_library_paths_win
+else:
+    setup_library_paths = lambda _1, _2: None
```

Comment:
I'm getting
```
Traceback (most recent call last):
  File "bin/workbench", line 11, in <module>
    load_entry_point('workbench', 'gui_scripts', 'workbench')()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 572, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2769, in load_entry_point
    return ep.load()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2422, in load
    return self.resolve()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2428, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/home/rwp/workbench-package-layout/qt/applications/workbench/workbench/app/mainwindow.py", line 45, in <module>
    plugins.setup_library_paths()
TypeError: <lambda>() missing 2 required positional arguments: '_1' and '_2'
```

I changed it to `setup_library_paths = lambda: None` to get it to work.